app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -66,12 +68,25 @@ ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope
       .catch(miqService.handleFailure);
   };
 
+  $scope.filterCloudNetworkChanged = function(id) {
+    miqService.sparkleOn();
+    $http.get('/network_router/network_router_subnets_by_network/' + id)
```

Comment:
@AparnaKarve,

I'm happy to use the API routes.
To be able to do so we need to add those requests to the API, correct?
Also I can't find any developer specific information about the API, do you have any pointers?
Also because there are a bunch of controller routes used by the networking items would not it make sense to refactor those in separate PRs?